{
    "type": "bundle",
    "id": "bundle--3cdea10b-707a-44de-b5e0-00d0b90cd89f",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--69e48711-6499-4563-b77d-e1544e3eaece",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-01-28T00:19:49.483258Z",
            "modified": "2025-01-28T00:19:49.483258Z",
            "name": "CVE-2025-24094",
            "description": "A race condition was addressed with additional validation. This issue is fixed in macOS Ventura 13.7.3, macOS Sequoia 15.3, macOS Sonoma 14.7.3. An app may be able to access user-sensitive data.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-24094"
                }
            ]
        }
    ]
}